src/components/button/button.tsx

```diff
@@ -241,8 +246,8 @@ export class Button
     );
 
     const contentEl = (
-      <span class={CSS.content}>
-        <slot />
+      <span class={CSS.content} ref={(el) => (this.contentEl = el)}>
+        <slot onSlotchange={this.handleSlotchange} />
```

Comment:
I'm not sure you need onSlotChange actually, a resize observer should be enough. The resize observer can just get the text from the childEl. Probably from the textContent property on the element.

Reply:
I thought we decided that there is no resize happening here for resizeObserver to pick up.

Reply:
I'm suggesting having a function to determine whether or not to show the title attribute which can be called initially in componentDidLoad and anytime a resize of the component occurs.

Someting like:

```ts
setFullLengthText = () :void => {
const {contentEl} = this;
this.fullLengthSlottedText = contentEl.offsetWidth < contentEl.scrollWidth ?
  contentEl.innerText : 
   null;
}
```

```
resizeObserver = createObserver("resize", () => this.setFullLengthText());
```

Reply:
Right, I understand.